Drop sync IPC messages in chrome/renderer.

suyambulingamrm opened the pull request:
IPC msgs like ChromeViewHostMsg_ContentBlocked,
ChromeViewHostMsg_AllowIndexedDB, ChromeViewHostMsg_AllowDOMStorage
are not handled in platforms like tizen. This leads to loading issues
on major websites, so temporarily dropping these messages.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

Chikdolman reviewed (approved):
In order to drop the msg, IPC Msgs should be investigated further in the future for its actual needs.
However, the change is needed for now for loading major webpages (such as naver, daum, etc).

nke94 reviewed (commented):
If possible, it will be better add message handler in platform side.
even though we cannot apply proper feature implement. why don't you add to send expected response message.

Commit:
Drop sync IPC messages in chrome/renderer.

IPC msgs like ChromeViewHostMsg_ContentBlocked,
ChromeViewHostMsg_AllowIndexedDB, ChromeViewHostMsg_AllowDOMStorage
are not handled in platforms like tizen. This leads to loading issues
on major websites, so temporarily dropping these messages.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

suyambulingamrm commented:
> If possible, it will be better add message handler in platform side.
> even though we cannot apply proper feature implement. why don't you add to send expected response message.

Yes sure. Now we are handling these messages.
